R/utils-mzr.R

```diff
@@ -1,6 +1,25 @@
 #' @include main.R
 NULL
 
+getHeaders <- function(path, rtRange, MSLevel, verbose = FALSE)
```

Comment:
Does it make sense to cache this? Probably not because it's fast? But still wanted to check :-)

Reply:
Is fast. Probably the parallel processing would be more benefitial than the cache. Also, I think this would change when other backend to parse data is integrated. I leave it to you ;)